index.bs

```diff
@@ -128,7 +123,7 @@ spec: webidl
       or other sources this specification hasn't anticipated.
     </dd>
 
-    <dt><dfn export>Powerful feature</dfn></dt>
+    <dt><dfn export lt="feature|features">Powerful feature</dfn></dt>
```

Comment:
In order for `<a>powerful feature</a>` to work, you have to include it in the |'ed list. Also, Bikeshed will automatically strip "s" from the end of a word when linking it, so you only need `lt="feature|powerful feature"`.